Components/Cards/cards.css

```diff
@@ -0,0 +1,209 @@
+/* cards  */
+
+.card {
+  background-color: var(--light-color);
+  min-width: 20rem;
+  max-width: 20rem;
+  max-height: 31.25rem;
+  display: flex;
+  flex-direction: column;
+  border-radius: 0.5rem;
+  margin: 0.5rem;
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+  position: relative;
+  transition: all 0.5s ease;
+}
+.card:hover {
+  cursor: pointer;
+  box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 28px,
+    rgba(0, 0, 0, 0.22) 0px 10px 10px;
+}
+
+/* overlay on card */
+.overlay-content {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
+.overlay-text {
+  font-size: 3rem;
+  font-weight: 700;
+}
+.card:hover .overlay-content {
+  opacity: 0.95;
+}
+
+/* card header */
+
+.card-header {
+  padding: 1rem 0.5rem 0.5rem;
```

Comment:
Don't use rem unit for the sizes that are less than 1rem

Reply:
Yeah. I have been using them meanwhile. I will try not to use it explicitaly.